go.mod

```diff
@@ -3,10 +3,11 @@ module github.com/labstack/echo/v4
 go 1.14
 
 require (
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
```

Comment:
The go.mod updates are very likely not required.
Please cleanup your PR to not modify unrelated files.

Reply:
Fixed!